bgpd/bgp_evpn_mh.c

```diff
@@ -1952,6 +2014,8 @@ static void bgp_evpn_es_show_entry(struct vty *vty,
 		char vtep_str[ES_VTEP_LIST_STR_SZ + BGP_EVPN_VTEPS_FLAG_STR_SZ];
 
 		type_str[0] = '\0';
+		if (es->flags & BGP_EVPNES_BYPASS)
```

Comment:
since we're adding a flag does the size of type_str need to grow by 1?

Reply:
Good catch, ack.

Reply:
I have increased BGP_EVPN_FLAG_STR_SZ to fit all bits and NULL.